README.md

```diff
@@ -43,6 +41,15 @@ As such, we recommend you install it in the following way.
    ```bash
    python tutorials/simple_example.py
    ```
+1. *(optional)* Check whether your local version of dockstring matches the dockstring dataset.
+   This is only necessary if you plan to mix docking scores from the dockstring dataset
```

Comment:
I would add

`This is only if you plan to mix PREVIOUSLY-COMPUTED docking scores from the docking dataset`